scripts/download/moby-openapi.mjs

```diff
@@ -13,4 +14,7 @@ export default async function run() {
   const outPath = path.join(process.cwd(), 'src', 'go', 'wsl-helper', 'pkg', 'dockerproxy', 'swagger.yaml');
 
   await download(url, outPath, { access: fs.constants.W_OK });
+
+  spawnSync('go', ['generate', '-x', './...'], { cwd: path.join(process.cwd(), 'src', 'go', 'wsl-helper') });
```

Comment:
I assume this will generate beyond just Moby API if there are more go generatable stuff in that path, correct?